libs/barista-components/chart/src/heatfield/chart-heatfield.scss

```diff
@@ -19,7 +19,25 @@ $animation-delay: 40ms;
   transition: transform $animation-duration ease-in-out $animation-delay;
 }
 
-.dt-chart-heatfield-marker.dt-chart-heatfield-active {
+.dt-chart-heatfield-marker .dt-chart-heatfield-text {
+  font-size: 6px;
```

Comment:
On an accessibility level, this font-size is definitely too small. Are there mocks / designs / requirements for this added feature?

Reply:
I felt the same, that's why I decided to include another Input to make the marker always expanded. the designs are here (ignore the blue border):
![image](https://user-images.githubusercontent.com/104001591/164299754-b636cc4c-3bf6-481f-a433-561db29e6291.png)
https://www.figma.com/file/5KrsvwWXq15BscXOWg4NrM/Charts?node-id=6145%3A172987